python_modules/dagster/dagster/_core/definitions/declarative_scheduling/asset_condition.py

```diff
@@ -512,17 +513,21 @@ class AssetConditionResult:
     start_timestamp: float
     end_timestamp: float
 
-    true_subset: AssetSubset
+    true_slice: AssetSlice
```

Comment:
We've discussed this, but I don't think we should use the term `true_slice`. Don't need to block this stack, but at some point will push on renaming this.